src/main/java/org/isf/examination/rest/ExaminationController.java

```diff
@@ -268,4 +208,42 @@ public ResponseEntity<List<PatientExaminationDTO>> getByPatientId(@PathVariable
 			return ResponseEntity.ok(listPatientExaminationDTO);
 		}
 	}
+	
+    public void validateExamination(PatientExaminationDTO newPatientExamination) throws OHServiceException {
+	ExaminationParameters.initialize();
+		
+	if (newPatientExamination.getPex_height() < ExaminationParameters.HEIGHT_MIN || newPatientExamination.getPex_height() > ExaminationParameters.HEIGHT_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("The size should be between "+ExaminationParameters.HEIGHT_MIN+" and "+ExaminationParameters.HEIGHT_MAX));
+        }
+        if (newPatientExamination.getPex_weight() < ExaminationParameters.WEIGHT_MIN || newPatientExamination.getPex_weight() > ExaminationParameters.WEIGHT_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("The weight should be between"+ExaminationParameters.WEIGHT_MIN+" and "+ExaminationParameters.WEIGHT_MAX));
+        }
```

Comment:
what if `null` values are passed?
Anyway:
```suggestion
	if (newPatientExamination.getPex_height() < ExaminationParameters.HEIGHT_MIN || newPatientExamination.getPex_height() > ExaminationParameters.HEIGHT_MAX) {
        	throw new OHAPIException(new OHExceptionMessage("The size should be between "+ExaminationParameters.HEIGHT_MIN+" and " + ExaminationParameters.HEIGHT_MAX));
        }
        if (newPatientExamination.getPex_weight() < ExaminationParameters.WEIGHT_MIN || newPatientExamination.getPex_weight() > ExaminationParameters.WEIGHT_MAX) {
        	throw new OHAPIException(new OHExceptionMessage("The weight should be between"+ExaminationParameters.WEIGHT_MIN+" and " + ExaminationParameters.WEIGHT_MAX));
        }
```